readme.txt

```diff
@@ -3,8 +3,8 @@ Contributors: automattic, nbachiyski, batmoo, johnjamesjacoby, philipjohn
 Tags: liveblog
 Requires at least: 4.4
 Requires PHP: 5.6
-Tested up to: 4.9.8
-Stable tag: 1.9.5
+Tested up to: 5.3.2
```

Comment:
Would be nice to have this tested on at least 5.7, if not a pre-5.8 release as well.

Reply:
@philipjohn Has it been tested on latest WP?

Reply:
It's now been tested with 5.8 so I've updated the tested tag in [cd8d219]